android/app/build.gradle

```diff
@@ -22,18 +22,18 @@ buildscript {
       googleFirebaseServicesDefault
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:8.4.1'
+    classpath 'com.android.tools.build:gradle:8.5.0'
```

Comment:
What's new?

If there is nothing specific we need asap, then I'd suggest to wait for a couple of patch releases and then update to `8.5.x`

Also upgrading AGP means all android devs will have to upgrade their AS.

Reply:
8.5.1 is available. Updated the PR.